### PR TITLE
When copr repo belongs to a group, alter the link

### PR DESCRIPTION
**Pull request by hroncok**
Fixes https://github.com/fedora-infra/fmn/issues/134

### CODE REVIEW

**Comment by hroncok** [comment]
I just edited this on GitHub, but it seems test are failing, so I'll look into that later

**Comment by pypingou** [comment]
This is the error:
````
  File "/home/travis/build/fedora-infra/fedmsg_meta_fedora_infrastructure/fedmsg_meta_fedora_infrastructure/coprs.py", line 132, in link
    if user.startswith('@'):
AttributeError: 'NoneType' object has no attribute 'startswith'
````
Should be easy to fix :)

**Comment by hroncok** [comment]
Thanks, I've seen it. I could only do it if it is a string... Not sure if that's the solution and why it is None in the first place.

**Comment by pypingou** [comment]
the simplest would be: ```  if user and user.startswith('@'):```

**Comment by hroncok** [comment]
You were seconds faster :)

**Comment by pypingou** [comment]
Looks good to me, let's merge! :)

**Comment by pypingou** [comment]
hm actually, it looks good but I think we should add unit-tests for it, to be on the safe side of things

**Comment by hroncok** [comment]
In the spirit of the original Pull Request, I'll try to provide test only by editing trough GitHub web interface.  Any struggles can later be squashed :)

**Comment by hroncok** [comment]
Seems to work.

File: fedmsg_meta_fedora_infrastructure/tests/coprs.py [hunk 1]
```diff
@@ -249,6 +249,39 @@ class TestCoprsWorkerCreate(Base):
     }
 
 
+class TestCoprsGroupLink(Base):
+    """ `Copr <https://fedorahosted.org/copr/>`_ publishes these messages
+    when a build has completed.  This is an example of a group repo.
+    """
+    expected_link = ("https://copr.fedoraproject.org/coprs/"
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/coprs.py]: Could we include the expected title as well?

Comment (hroncok) [inline fedmsg_meta_fedora_infrastructure/tests/coprs.py]: I don't know what that's expected to be. `copr.build.end`?

Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/coprs.py]: Indeed, the title isn't really informative, the subtitle is though: 
https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/blob/develop/fedmsg_meta_fedora_infrastructure/tests/coprs.py#L103-L104

Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/coprs.py]: So should we add the subtitle?

**Comment by hroncok** [comment]
Travis build is broken not by my change I believe.

File: fedmsg_meta_fedora_infrastructure/tests/coprs.py (hunk 1 above)
Comment (hroncok) [inline fedmsg_meta_fedora_infrastructure/tests/coprs.py]: done

**Comment by pypingou** [comment]
> error: Setup script exited with error in urllib3 setup command: Invalid environment marker: python_version <= "2.7"

...

> [31;1mThe command "python setup.py install" failed and exited with 1 during .[0m

Bummer :(

**Comment by hroncok** [comment]
Last commit fixes the issue.

**Comment by pypingou** [comment]
Cool! Looks good to me, let's merge :)

**Pull request merged**